juntagrico/entity/member.py

```diff
@@ -80,14 +74,43 @@ def usable_shares(self):
     def usable_shares_count(self):
         return self.usable_shares.count()
 
+    @property
+    def subscription_future(self):
+        sub_membership = self.subscriptionmembership_set.filter(subscription__activation_date__isnull=True).first()
```

Comment:
I would check here for "`join_date` is set and not in the future"
And on the other side assert in general, that join_date >= activation_date.